docs/migration_guide.rst

```diff
@@ -1,11 +1,29 @@
 Migration guide
 ===============
 
+**********************
+0.19.0 Migration guide
+**********************
+
+0.19.0 Breaking changes
+-----------------------
+
+.. currentmodule:: starknet_py.net.client_models
+
+1. :class:`GatewayClient` and all related classes have been removed.
+2. Client ``net`` property has been removed.
+3. :class:`TransactionReceipt` field ``execution_resources`` has been changed from ``dict`` to :class:`ExecutionResources`.
```

Comment:
Maybe we should additionally have info about fields removed from `TransactionReceipt`?

Alternatively: 
> 1. :class:`GatewayClient` and all related classes and fields have been removed.

Reply:
Changed to your suggestion. There will be more information added to the migration guide after models refactoring that will take place before next release.